tests/test_finder.py

```diff
@@ -83,7 +83,7 @@ def _verify_and_get_dumps(self, count, index=1, multistep=False):
             self.assertLessEqual(len(first_steps), len(steps))
         return dumps
 
-    def _verify_dumped_images(self, needle_name, haystack_name, dumps, backend):
+    def _verify_dumped_images(self, needle_name: str, haystack_name: str, dumps: str, backend: str) -> None:
```

Comment:
Let's just use `Any` or not document this right now.

Reply:
so I just leave it undocumented.

Reply:
Yes, it is good enough for the unit tests for now and will be updated once we work on those